src/content/structured/components/section-container/code.mdx

```diff
@@ -0,0 +1,131 @@
+---
+path: "/components/section-container/code"
+
+date: "2022-09-30"
+
+title: "Section container"
+
+status: "BETA"
+
+subTitle: "The section container component is used to consistently align an app's content."
+
+contribute: "https://github.com/mi6/ic-design-system/tree/main/src/content/structured/components/section-container/code.mdx"
+
+hidden: false
+
+tabs:
+  [
+    { title: "Guidance", path: "/components/section-container" },
+    { title: "Code", path: "/components/section-container/code" },
+  ]
+---
+
+## Component demo
+
+import { IcSectionContainer } from "@ukic/react";
+
+export const snippetsDefault = [
+  {
+    language: "Web component",
+    snippet: `<ic-section-container>
```

Comment:
something weird going on with the spacing of all the snippets: 
![image](https://user-images.githubusercontent.com/116102637/208466987-41832cf4-37b6-4280-af38-340bc0b5efc2.png)